src/server.js

```diff
@@ -125,19 +135,27 @@ const getCompletionItems = (params) => {
           const item = CompletionItem.create(name)
 
           item.kind = CompletionItemKind.Color // Use Text instead of Color
-          let markdown = `| Theme | Color |
-| --- | --- |`
-          allColors.forEach(([themeName, hsl]) => {
+          let markdown = `| Theme | Color | Value |
+| --- | --- | --- |`
+          allColors.forEach(([themeName, value]) => {
             // ignore components like dark_Button
             const ignorePattern = /_[A-Z]/
 
             if (ignorePattern.test(themeName)) {
               return
             }
 
-            const svg = `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24"><rect x="1" y="1" width="22" height="22" fill="${hsl}" rx="4" /></svg>`
+            const isAlphaFunction = value.startsWith("rgba(") || value.startsWith("hsla(")
```

Comment:
would be nice to extract this into a little function with some tests

Reply:
Yeah, this probably deserves a test.

I think in order to properly extract this into its own testable unit, the code for the server portion of the extension probably should be split from the client code so that it can be organized into modular pieces.

Happy to help with that effort if desired - just didn't want to throw a giant refactoring diff at you without discussing it first. :sweat_smile:

Reply:
what if we just used like `color` and check if the alpha isn’t 1? i think it’s already installed in the repo

Reply:
> what if we just used like `color` and check if the alpha isn’t 1? i think it’s already installed in the repo

Oh nice, didn't see that was there. I just updated this PR to use it instead. 

Also pulled in the "skip id key" fix from #2 because `color` wasn't happy about parsing theme names as color values.